guides/source/asset_pipeline.md

```diff
@@ -33,16 +33,17 @@ passing the `--skip-sprockets` option.
 $ rails new appname --skip-sprockets
 ```
 
-Rails automatically adds the `sass-rails` gem to your `Gemfile`, which is used
-by Sprockets for asset compression:
+Rails automatically adds the [`sass-rails`](https://github.com/rails/sass-rails)
+gem to your `Gemfile`, which is used by Sprockets for
+[Sass](https://sass-lang.com) compilation:
 
 ```ruby
 gem 'sass-rails'
 ```
 
 Using the `--skip-sprockets` option will prevent Rails from adding
-them to your `Gemfile`, so if you later want to enable
-the asset pipeline you will have to add those gems to your `Gemfile`. Also,
+this to your `Gemfile`, so if you later want to enable
+the asset pipeline you will have to add that gem to your `Gemfile`. Also,
```

Comment:
> Rails automatically **adds the `sass-rails` gem to your `Gemfile`** [...] Using the `--skip-sprockets` option will prevent Rails from **adding this to your `Gemfile`**, so if you later want to enable the asset pipeline you will have to **add that gem to your `Gemfile`**.

This reminds me of:

["By installing Java, you will be able to experience the power of Java."](https://www.joelonsoftware.com/2009/01/12/by-installing-java-you-will-be-able-to-experience-the-power-of-java/) 😄 

Perhaps:

```suggestion
this gem, so if you later want to enable the asset pipeline
you will have to add it to your `Gemfile` manually. Also,
```

Reply:
👍 I've applied your suggestion in fd9a032eea94371cd811c149f3b5f0e64ec0ee69